packages/itwinui-css/backstop/scenarios/input.js

```diff
@@ -13,4 +13,20 @@ module.exports = [
   scenario('Type Without Label', {
     selectors: ['#demo-no-label'],
   }),
+  scenario('State hover in default', {
+    actions: [hover('#test-input')],
+    selectors: ['#demo-input'],
+  }),
+  scenario('State hover with status', {
+    actions: [hover('#test-input-with-status')],
+    selectors: ['#demo-input-with-status'],
+  }),
+  scenario('State focus in default', {
```

Comment:
You can reduce the amount of test images by covering hover & focus with a single test.  Here's an example from `carousel.js`:
```js
// Hover & focus states
scenario('State hover & focus', {
  actions: [hover('#test-1'), focus('#test-2')],
  selectors: ['#demo-default'],
}),
```

Reply:
Since multiple `iui-input-container` elements aren't wrapped in another element that isn't the entire demo page, I think it's best to just have separate test images for focus and hover